source/Octopus.Tentacle.Client/Retries/TaskExtensionMethods.cs

```diff
@@ -23,5 +25,43 @@ public static void IgnoreUnobservedExceptions(this Task task)
                 },
                 TaskContinuationOptions.OnlyOnFaulted | TaskContinuationOptions.ExecuteSynchronously);
         }
+
+        public static async Task<TaskCompletionResult> WaitTillCompletion(this Task actionTask, TimeSpan abandonAfter, CancellationToken cancellationToken)
+        {
+            var actionTaskCompleted = await actionTask.WaitTillCompletedOrCancelled(cancellationToken);
+
+            if (actionTaskCompleted) return TaskCompletionResult.Completed;
+
+            actionTaskCompleted = await actionTask.WaitTillCompletedOrTimeout(abandonAfter).ConfigureAwait(false);
```

Comment:
nit ConfigureAwait is sometimes used. Should we go consistent and use it or don't

Reply:
Yeah, I'm still not 100% sure what our policy on this is.

I think that we do not use it, right?

Either way, I've removed `ConfigureAwait` from this file only.

Reply:
Yeh I think we went with we don't need it